src/testUtils/mocks/entities.ts

```diff
@@ -501,6 +507,18 @@ export function getSecurityTokenTransfersAreFrozenStub(frozen: boolean): SinonSt
   return securityTokenTransfersAreFrozenStub;
 }
 
+/**
+ * @hidden
+ * Retrieve the stub of the `SecurityToken.Transfers.canTransfer` method
+ */
+export function getSecurityTokenTransfersCanTransferStub(status: TransferStatus): SinonStub {
```

Comment:
status should be optional

Reply:
and so should `frozen` in the above getter, and the if should check for equality with undefined